Shared/UI/Views/MessageContentView/MessageContentView.swift

```diff
@@ -314,21 +314,24 @@ class MessageContentView: BaseView {
         self.loadTask?.cancel()
         
         self.loadTask = Task { [weak self] in
-            guard let `self` = self else { return }
```

Comment:
Removing this. This guard will create a strong reference throughout the life of the task. We don't really want that if the task is view removed before the task completes.